devfile.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: 1.0.0
+metadata:
+  name: jbang
+projects:
+  -
+    name: jbang
+    source:
+      type: git
+      location: "https://github.com/maxandersen/jbang.git"
```

Comment:
this means forks wont use their own repo for edits - can we avoid that ?

Reply:
I suppose the fork low you describe is not supported
@skabashnyuk  could you please confirm?
But in any case having a "Try it on Hosted Che" button for the original project is the step forward IMO.

Reply:
fixed. Starting from 7.8.0 upstream version it is possible not to specify the project

Reply:
awesome! do you know when this is available on che.openshift.io ?

Reply:
@maxandersen it is already available, you can try via the button in the PR description